src/app/api/services/dot-license/dot-license.service.ts

```diff
@@ -13,10 +23,12 @@ export class DotLicenseService {
      * @memberof DotLicenseService
      */
     isEnterpriseLicense(): Observable<boolean> {
-        return this.dotcmsConfig
-            .getConfig()
-            .take(1)
-            .pluck('license')
+        return this.coreWebService
+            .requestView({
+                method: RequestMethod.Get,
+                url: this.configUrl
+            })
```

Comment:
Maybe we should have a 'getLicense' method here and do into it the request (maybe it should be private at least for now), later we could change the 'getLicense' method for not do a request every time when we'll have a web socket event